velox/docs/functions/spark/datetime.rst

```diff
@@ -200,6 +200,23 @@ These functions support TIMESTAMP and DATE input types.
         SELECT make_ym_interval(2); -- 2-0
         SELECT make_ym_interval(); -- 0-0
 
+.. spark:function:: make_dt_interval([days[, hours[, minutes[, seconds]]]]) -> interval day to second
+
+    Make day-time interval from ``days``, ``hours``, ``minutes`` and ``seconds`` fields.
+    All parameters can be zero, positive or negative.
+    Throws an error when inputs lead to int overflow.
```

Comment:
I assume not all parameters are integer, perhaps clarify the parameter types and the overflow behavior.